src/components/Table.jsx

```diff
@@ -103,13 +131,14 @@ export default function Table() {
 
     fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}&unique=status`, {
       method: "POST",
+      body: JSON.stringify(body),
       headers: {
         "Content-type": "application/json; charset=UTF-8"
       }
     })
       .then((response) => response.json())
       .then((result) => setStatus(result?.data?.uniqueValues));
```

Comment:
same as above. 
use await and handle errors.

Reply:
addressed